README.md

```diff
@@ -1 +1,12 @@
-# proposals
\ No newline at end of file
+# Stencil Community Proposals
+
+Use this repository to create proposals for new Stencil Community packages.
+
+## Usage
+
+Proposals are written in issues.
+
+To create a new proposal,
+
+1. Search for existing proposals in [the issues](https://github.com/stencil-community/proposals/issues). The proposal may exist already. If it does, consider reading through it and adding your thoughts. You can also react to proposals by adding a :+1: reaction.
+1. If it does not exist, [create a new issue](https://github.com/stencil-community/proposals/issues/new) and fill out the template. :sparkles:
```

Comment:
Discussed offline about creating these templates

Reply:
Template added in [f1b541d](https://github.com/stencil-community/proposals/pull/1/commits/f1b541dd6c6f1aedf565ad963b5a3d6547de9630)